### DISCUSSION TITLE
ST-link v3 Debugger Connector for Debug Adapter

### DISCUSSION THREAD

**Discussion by dcherenson** [bitcraze/discussions#471] [Q&A] [answered]
I wanted to know if [this ST Link v3 ](https://www.mouser.com/ProductDetail/STMicroelectronics/STLINK-V3MINIE?qs=MyNHzdoqoQKcLQe5Jawcgw%3D%3D&mgh=1&gclid=CjwKCAiAheacBhB8EiwAItVO2-NoaMF0ofl9uYM7Gu2DAaaMm7YWlniAP-uhO8_PLBH8r6TjfPTp5xoCPUYQAvD_BwE)would work with the debugger adapter. I could not find the exact requirements for the debugger on this [page](https://www.bitcraze.io/documentation/repository/crazyflie-firmware/master/development/openocd_gdb_debugging/), only "ST Link V2 or similar"

It seems like it's hard to find a V2 and this one is a lot cheaper. Thank you

**Comment by ataffanel** [comment, marked as answer]
The debug adapter has the same 10-pin debug connector as the stlink-v3 you linked, so as long as you have the right cable it should work find.

Note that latest version of the ST-Link are not compatible with anything else than ST chips so this will work fine for the Crazyflie's STM32F4 but not for the nRF51 (not out of the box at least).

**Reply by dcherenson** [reply]
Thank you!
